src/components/survey/Survey.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
-import { QuestionGroup, Question } from '../question/Question';
+import { QuestionGroup, Question, QuestionGroupList } from '../question/Question';
 
-export default function SurveyGenerator() {
+export interface Survey {
+  id: number;
+  owner: string;
+  title: string;
+  description: string;
+  questionCount: number;
+  image?: string;
+  status?: "NEW" | "VIEWED" | "SOLVED";
+}
```

Comment:
If you did not create this `interface Survey` only for the timeline-page then you need to add `questions` in it otherwise you should change its name to something like `SurveyTimeline`. 
```suggestion
export enum SurveyStatus {
  NEW,
  VIEWED.
  SOLVED
}
export interface SurveyTimeline {
  id: number;
  owner: string;
  title: string;
  description: string;
  questionCount: number;
  image?: string;
  status?: SurveyStatus;
}
```